pom.xml

```diff
@@ -21,7 +21,7 @@
   <properties>
     <revision>1.52</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.222.1</jenkins.version>
+    <jenkins.version>2.249.1</jenkins.version>
```

Comment:
Is this still needed?

Reply:
Yes the permissions field on layout.jelly either wasn’t implemented on this version or didn’t work. Bumping the baseline fixed it. I just picked the oldest version in bom, didn’t try and track down the change which made this work